plugins/examples/settings.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+include ':custom-settings'
+include ':custom-significance-heuristic'
+include ':custom-suggester'
+include ':painless-whitelist'
+include ':rescore'
+include ':rest-handler'
+include ':script-expert-scoring'
+include ':security-authorization-engine'
```

Comment:
Can we scan for immediate subdirectories with a build.gradle, similar to what we do in the root project settings.gradle?

Reply:
We _could_ but given this is an "example" project I think it's best to avoid as much "magic" as possible here.

Reply:
The root isn’t really the example, though. Each subdirectory is its own sample project

Reply:
True. I've gone ahead and made this dynamic.